api/controllers/game/editmode.js

```diff
@@ -0,0 +1,27 @@
+module.exports = function (req, res) {
+  const { isRanked } = req.body;
+  const promiseGame = gameService.findGame({ gameId: req.session.game });
+  const promisePlayer = userService.findUser({ userId: req.session.usr });
+  Promise.all([promiseGame, promisePlayer])
+    .then(function changeAndSave(values) {
+      const [game, player] = values;
+      const gameUpdates = {};
+
+      gameUpdates.isRanked = isRanked;
```

Comment:
Do we modify `gameUpdates` anywhere else or can we one line this?

`const gameUpdates = {isRanked};`